includes/data-stores/class-wc-admin-reports-orders-data-store.php

```diff
@@ -279,6 +283,17 @@ public function get_data( $query_args ) {
 				return new WP_Error( 'woocommerce_reports_revenue_result_failed', __( 'Sorry, fetching revenue data failed.', 'wc-admin' ) );
 			}
 
+			$unique_products = $wpdb->get_var(
+				"SELECT
+						COUNT( DISTINCT {$wpdb->prefix}wc_order_product_lookup.product_id )
+					FROM
+						{$wpdb->prefix}wc_order_product_lookup JOIN wp_posts ON {$wpdb->prefix}wc_order_product_lookup.order_id = wp_posts.ID
```

Comment:
`wp_posts` should be updated to `{$wpdb->prefix}posts`

Reply:
Fixed!